example/src/main/java/com/capgemini/mrchecker/selenium/pages/projectY/ExitIntentPage.java

```diff
@@ -1,47 +1,47 @@
 package com.capgemini.mrchecker.selenium.pages.projectY;
 
-import com.capgemini.mrchecker.selenium.core.BasePage;
-import com.capgemini.mrchecker.selenium.pages.environment.GetEnvironmentParam;
-import com.capgemini.mrchecker.selenium.pages.environment.PageSubURLsProjectYEnum;
-import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.math.BigDecimal;
+import java.util.concurrent.TimeUnit;
+
 import org.openqa.selenium.By;
 import org.openqa.selenium.Dimension;
 import org.openqa.selenium.support.pagefactory.ByChained;
 
-import java.awt.*;
-import java.math.BigDecimal;
-import java.util.concurrent.TimeUnit;
+import com.capgemini.mrchecker.selenium.core.BasePage;
+import com.capgemini.mrchecker.selenium.pages.environment.GetEnvironmentParam;
+import com.capgemini.mrchecker.selenium.pages.environment.PageSubURLsProjectYEnum;
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
 
 public class ExitIntentPage extends BasePage {
-
-	private static final String MODAL_WINDOW_HIDDEN           = "display: none;";
-	private static final String MODAL_WINDOW_DISPLAYED        = "display: block;";
-	private static final String MODAL_WINDOW_STYLE_ATTRIBUTTE = "style";
-
-	private static final By selectorModalWindow            = By.cssSelector("div#ouibounce-modal");
-	private static final By selectorExitIntentText         = By.cssSelector("div#content h3");
-	private static final By selectorModalWindowTitle       = By.cssSelector("h3");
-	private static final By selectorModalWindowCloseButton = By.cssSelector("div.modal-footer > p");
-
+	private static final String	MODAL_WINDOW_DISPLAYED			= "display: block;";
```

Comment:
Weird formatter behavior. Please confirm that you've used it here. Are you using IntelliJ or Eclipse?